Gemfile

```diff
@@ -22,6 +22,9 @@ gem "discard", "~> 1.0"
 gem "dotenv-rails"
 gem "factory_bot_rails", "~> 4.8", require: false
 gem "faker", require: false
+gem "flipper"
```

Comment:
If I get this right, `FeatureToggled.enabled?` becomes `Flipper.enabled?` everywhere?

Reply:
Yup. Once we ship this, we can go create Flipper flags for all our feature flags, and then open a PR that migrates to using `Flipper.enabled?`. Any new feature flags we introduce in the meantime can start using Flipper straight away.

We can also start doing things like `Flipper.enabled?(feature, current_user)`, ie, target specific user groups to soft-launch features to.